shiny/_fileupload.py

```diff
@@ -52,7 +52,8 @@ def file_begin(self) -> None:
         file_info["datapath"] = os.path.join(
             self._dir, str(self._n_uploaded) + file_ext
         )
-        self._current_file_obj = open(file_info["datapath"], "ab")
+        with open(file_info["datapath"], "ab") as f:
+            self._current_file_obj = f
```

Comment:
@fpgmaas This is the change that looked dangerous. I think it'll close the file handle after it exits the `with`, so `self._current_file_obj` will not be usable.

Reply:
@wch My bad; this is not a change `isort` made, but a change resulting by me adding a `flake8` extension and not properly solving the issue.. I have opened a MR that only shows the difference in non-`.py` files. When I have received feedback there and I have processed the feedback, I will add a single commit with only the output of `isort .` applied.